app/controllers/posts_controller.rb

```diff
@@ -2,26 +2,35 @@ class PostsController < ApplicationController
   # TODO (Shimmy): Define logged_in_user
   # before_action :logged_in_user, only: [:create, :destroy]
 
+  def index
+    @group = Group.friendly.find(params[:group_id])
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  # TODO (Shimmy): Watch out for duplicates
+  # TODO (Shimmy): Flash success notification on send
   def create
     @group = Group.friendly.find(params[:group_id])
     @post = @group.posts.build(post_params)
-    @post.user_id = current_user.id
-    if @post.save
-      redirect_to group_path(@group), notice: "Post has been saved successfully."
-    end
+    @post.save
```

Comment:
@negativetwelve 

 `@post` is nil here. Unable to use `@post.create(post_params)`. I think this is because the `@post`  is actually created in the GroupsController (https://github.com/calblueprint/ashby-village/blob/master/app/controllers/groups_controller.rb#L10). Is this bad practice since it seems more logical to be made in the PostsController's new method? 

I removed the index and new method in the PostsController because they weren't being used at all.